data/i18n/en/en.toml

```
@@ -148,30 +148,102 @@ other = "Latest Release:"
 [latest_version]
 other = "latest version."
 
-[layouts_blog_pager_prev]
-other = "<< Prev"
+[layout_docs_glossary_architecture_description]
+other = "The inner components of Kubernetes."
+
+[layout_docs_glossary_architecture_name]
+other = "Architecture"
+
+[layout_docs_glossary_community_description]
+other = "Related to Kubernetes open-source development."
+
+[layout_docs_glossary_community_name]
+other = "Community"
+
+[layout_docs_glossary_core-object_description]
+other = "A resource type that Kubernetes supports by default."
+
+[layout_docs_glossary_core-object_name]
+other = "Core Object"
+
+[layout_docs_glossary_extension_description]
+other = "Supported customizations of Kubernetes."
+
+[layout_docs_glossary_extension_name]
+other = "Extension"
+
+[layout_docs_glossary_fundamental_description]
+other = "Relevant for a first-time user of Kubernetes."
+
+[layout_docs_glossary_fundamental_name]
+other = "Fundamental"
+
+[layout_docs_glossary_networking_description]
+other = "How Kubernetes components talk to each other (and to programs outside the cluster)."
+
+[layout_docs_glossary_networking_name]
+other = "Networking"
+
+[layout_docs_glossary_operation_description]
+other = "Starting and maintaining Kubernetes."
+
+[layout_docs_glossary_operation_name]
+other = "Operation"
+
+[layout_docs_glossary_security_description]
+other = "Keeping Kubernetes applications safe and secure."
+
+[layout_docs_glossary_security_name]
+other = "Security"
+
+[layout_docs_glossary_storage_description]
+other = "How Kubernetes applications handle persistent data."
+
+[layout_docs_glossary_storage_name]
+other = "Storage"
+
+[layout_docs_glossary_tool_description]
+other = "Software that makes Kubernetes easier or better to use."
+
+[layout_docs_glossary_tool_name]
+other = "Tool"
+
+[layout_docs_glossary_user-type_description]
+other = "Represents a common type of Kubernetes user."
+
+[layout_docs_glossary_user-type_name]
+other = "User Type"
+
+[layout_docs_glossary_workload_description]
+other = "Applications running on Kubernetes."
+
+[layout_docs_glossary_workload_name]
+other = "Workload"
 
 [layouts_blog_pager_next]
```

This should be moved up along with layouts_blog_pager_prev to the correct location BEFORE layout_docs_*
```suggestion
[layouts_blog_pager_next]
```

---

layouts_blog_pager_prev comes after layout_docs_*

Should I change them to all to 'layouts' and update the code accordingly?

---

I wouldn't do that change in _this_ PR, as it's a tricky change to get right. We'd want to coordinate with multiple localization teams and it will take several weeks _minimum_. However, please **do** consider filing an issue about how the two prefixes don't match, so we can track the work.